src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const todoList = [
+  {
+    id: "1",
+    title: "Complete Assignment",
+   },
+   {
+     id: "2",
+     title: "Read",
+   },
+   {
+     id: "3",
+     title: "Be active",
+   },
+];
+
+function App() {
+  return (
+    <div style={{ textAlign: 'left' }}>
+      <header>
+        <p>
+          <h1>Todo</h1>
+          <ul>
+            {todoList.map(function(item) {
+              return (
+                <li key={item.objectID}>
```

Comment:
your item doesn't has an `objectID` key - in this case it will return `undefine`
it has only `id` and `title` keys
you want `key={item.id}`